modules/stereo/tutorials/quasi_dense_stereo.markdown

```diff
@@ -0,0 +1,55 @@
+Quasi dense Stereo {#quasi_dense_stereo}
+==================
+
+Goal
+----
+
+In this tutorial you will learn how to
+
+-   Configure a QuasiDenseStero object
+-   Compute dense Stereo correspondences.
+
+-----------
+[Source Code](../samples/dense_disparity.cpp)
```

Comment:
Link is broken:  http://pullrequest.opencv.org/buildbot/export/pr_contrib/1941/docs/d3/d7d/quasi_dense_stereo.html

Use @include or @snippet or GitHub link (opencv/opencv master branch)